build/common-config.sh

```diff
@@ -96,6 +100,21 @@ var_set() {
 	"
 	env_key_sort[$i]="PDO_ENCLAVE_CODE_SIGN_PEM"; i=$i+1; export PDO_ENCLAVE_CODE_SIGN_PEM=${env_val[PDO_ENCLAVE_CODE_SIGN_PEM]}
 
+    env_val[PDO_IAS_ROOT_CA_CERT_PATH]="${PDO_IAS_ROOT_CA_CERT_PATH:-${PDO_SGX_KEY_ROOT}/ias_root_ca.cert}"
+    env_desc[PDO_IAS_ROOT_CA_CERT_PATH]="
+        The path of the IAS report signing CA certificate. This is the self-signed certificate of IAS, which
+        certifies the attestation verification key.
+    "
+    env_key_sort[$i]="PDO_IAS_ROOT_CA_CERT_PATH"; i=$i+1; export PDO_IAS_ROOT_CA_CERT_PATH=${env_val[PDO_IAS_ROOT_CA_CERT_PATH]}
```

Comment:
is there any reason why an installation would put this any place other than SGX_KEY_ROOT?

Reply:
Not that I can think of.